application-services/custom/camera-management/appcamera/evam.go

```diff
@@ -218,6 +222,99 @@ func (app *CameraManagementApp) getPipelineStatus(deviceName string) (interface{
 	return nil, nil
 }
 
+// processEdgeXEvent is the function that is called when an EdgeX event is received
+func (app *CameraManagementApp) processEdgeXEvent(_ interfaces.AppFunctionContext, data interface{}) (bool, interface{}) {
```

Comment:
More explicit name since there are Events and SystemEvent in EdgeX, and this is specifically looking for Device System Events.

```suggestion
func (app *CameraManagementApp) processEdgeXDeviceSystemEvent(_ interfaces.AppFunctionContext, data interface{}) (bool, interface{}) {
```